src/hooks/useERC20Permit.ts

```diff
@@ -114,7 +114,7 @@ const PERMIT_ALLOWED_TYPE = [
   { name: 'allowed', type: 'bool' },
 ]
 
-export function useERC20Permit(
+function useERC20Permit(
```

Comment:
many of these are removed probably because they dont use them and maybe it's more optimised? tho tree shaking should take care of that. will re-add them regardless tho